### DISCUSSION TITLE
피그마 확인해주시고 의견 부탁드립니다

### DISCUSSION THREAD

**Discussion by orihy97** [tutorLink-SYU-Capstone2023/Front_end#5] [General]
https://www.figma.com/file/UxwLp3Zfm5aAEs63k28clf/5%ED%8C%80%EC%9E%85%EB%8B%88%EB%8B%B9?type=design&node-id=0-1&mode=design&t=VvBkx9HBS9Uoc464-0

**Comment by kmc6381** [comment]
프레임을 공통으로 맞추고 틀도 처음과 끝을 공통적이게 맞추면 좋을 것 같아요!

**Reply by orihy97** [reply]
### 의견 감사합니다
